SU-SWS/stanford_paragraph_types
# Reusable update functions to change machine names of fields on paragraph

# READY FOR REVIEW

# Summary
- Move field data from field_p_wysiwyg to field_p_callout_wysiwyg.

# Needed By (Date)
- End of sprint.

# Urgency
- mild

# Steps to Test

1. create complex page with a callout text.
2. checkout branch & update db
3. verify your complex page still looks the same.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
